hdfs3/lib.py

```diff
@@ -47,6 +49,14 @@ class EncryptionFileInfo(ct.Structure):
                 ('mIv', ct.c_char_p),
                 ('zone_key_version_name', ct.c_char_p)]
 
+    def to_dict(self):
+        return {'suite': self.suite,
+                'protocol_version': self.protocol_version,
+                'key': self.key,
+                'key_name': ensure_string(self.key_name),
+                'mIv': self.mIv,
+                'zone_key_version_name': self.zone_key_version_name}
+
 
 class EncryptionZoneInfo(ct.Structure):
     _fileds_ = [('suite', ct.c_int),
```

Comment:
Are you sure about `_fileds_`? :-)

Reply:
Good catch, but not related to our failure here. Fixed.